manager/apis/app/v1alpha1/fybrikapplication_types.go

```diff
@@ -189,6 +197,14 @@ type FybrikApplicationStatus struct {
 	// +optional
 	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
 
+	// ValidatedGeneration is the version of the FyrbikApplication that has been validated with the taxonomy defined.
```

Comment:
@eletonia Somehow after the merge the new structures appear twice.

Reply:
I just removed the duplicate variables in the latest commit